### ISSUE TITLE
Radio button and checkbox issues

### ISSUE THREAD

**Issue by maya** [18F/web-design-standards#1929]
1. When I placed the radio buttons and checkboxes in a three column layout, I experienced some of the form elements overlapping with other form elements in the adjacent column. This was not good bc I couldn't select the intended form element.
2. There is not a pointer style cursor (hand) when I hover over the elements, so it's not obvious where someone can click.
3. The input does not have the proper dimensions, since the label is stacked above the input and is clickable, it's generally not an issue, but this caused some issues (#1).
<img width="520" alt="screen shot 2017-05-15 at 5 18 05 pm" src="https://cloud.githubusercontent.com/assets/5249443/26084653/81ef7faa-3992-11e7-954e-a1869ccdf0fa.png">

I was able to fix these issues with the following code: https://github.com/18F/clinical-trials-prototype/blob/clean/_sass/_custom.scss#L183-L194

**Comment by toolness** [comment]
@maya I think the link you provided to your custom fixes is now out-of-date because that file has been changed since you created the link... Is this the commit that fixed things on your end?

https://github.com/18F/clinical-trials-prototype/commit/9bc2d20a697968eacdd94a1d12f09c38f453910e

**Comment by donjo** [comment]
Fixed in #2002
